recipes/pandas-compat/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.1.0" %}
+{% set name = "pandas-compat" %}
+{% set sha256 = "3f17b42567d1fda58d7af05a4ee5afb6c012e9337e032594f904150ddd691385" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - pandas
+
+test:
+  imports:
+    - pandas_compat
+
+about:
+  home: https://github.com/pandas-dev/pandas-compat
+  license: BSD 3-clause
```

Comment:
I think it is not unfortunately.

ref: https://github.com/pandas-dev/pandas-compat/commit/9e6d0d145a635cf280a8da8cc02ab0ffc30f3cde#diff-97c91a104c431d0c365565d3ac03ac13R3